lib/instagram_graph_api/client/insights.rb

```diff
@@ -0,0 +1,14 @@
+module InstagramGraphApi
+  class Client
+    module Insights
+      def insights(object_id, metrics: nil)
```

Comment:
@himrock922 We will have to use type for `media` context which is not available here, also we will need the `METRIC_HASH` here.